packages/hdwallet-xdefi/src/xdefi.ts

```diff
@@ -11,7 +11,7 @@ class XDeFiTransport extends core.Transport {
 }
 
 export function isXDeFi(wallet: core.HDWallet): wallet is XDeFiHDWallet {
-  return _.isObject(wallet) && (wallet as any)._isXDeFi;
+  return isObject(wallet) && (wallet as any)._isXDeFi;
```

Comment:
Nitpick: If we're going to import some lodash util, might as well use `_.has()` which will handle the non-object cases

Reply:
Good point -- though IMO consistency with the rest of the wallet's implementation of this type of guard is valuable, and the endgame for them should just be to nuke them wholesale (see #466 / #435).